packages/botonic-nlp/tests/tasks/ner/process/ner-sample-processor.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Codifier } from '../../../../src/preprocess/codifier'
+import { Preprocessor } from '../../../../src/preprocess/preprocessor'
+import { NerSampleProcessor } from '../../../../src/tasks/ner/process/ner-sample-processor'
+
+describe('NER Sample Processor', () => {
+  test('Process sample', () => {
+    const processor = new NerSampleProcessor(
+      new Preprocessor('en', 12),
+      new Codifier(
+        [
+          '<PAD>',
```

Comment:
too long for a test. create a testHelper method which returns the vocabulary or already the Codifier. Maybe call it testFashionCodifier/Vocabulary?